components/search/search.tsx

```diff
@@ -200,6 +252,7 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         if (!searchType) {
             setDropdownFocused(false);
         }
+        props.actions.getRecentSearches();
```

Comment:
I think this needs to be moved to happen when the user focuses the search box. I was testing this locally, and I was wondering why I wasn't seeing any search results, but it was because I just started typing without clicking on "Messages" or "Files"

Reply:
I see! it makes sense, thanks!